typescript/test/services/deposits.test.ts

```diff
@@ -670,11 +678,196 @@ describe("Deposits", () => {
   })
 
   describe("Deposit", () => {
-    // TODO: Implement unit tests for other functions.
+    describe("getBitcoinAddress", () => {
```

Comment:
Let's also add a case for mainnet. This can be done by switching the result returned from `MockBitcoinClient.getNetwork()`.